vpoly.s

```diff
@@ -215,6 +215,7 @@
 # current accumulated vector state: v1, v2, v3, v4, v5
 vector_poly1305:
 	# save registers
+	addi sp, sp, -96
```

Comment:
Can you omit these new calls and just make the store and loads use negative offsets? Seems more fitting with the decreasing stack.